### ISSUE TITLE
Wrong release name output for `version` command

### ISSUE THREAD

**Issue by jchristgit** [ausimian/forecastle#1]
When running the generated unversioned release script:

```sh
$ _build/dev/rel/bolt/bin/bolt 
Usage: bolt COMMAND [ARGS]
```

the name is correct. But when I run `version`, I get `castle_test`:

```sh
$ _build/dev/rel/bolt/bin/bolt version
castle_test 0.13.0
```

This should output my release's name instead, shouldn't it?

**Comment by ausimian** [comment]
Definitely seems like a bug! Thx for reporting, I’ll take a look at it later.

**Comment by ausimian** [comment]
This should be fixed now in [0.1.2](https://hex.pm/packages/forecastle/0.1.2). It should be sufficient for you to `mix deps.update forecastle` and regenerate your release.

**Comment by jchristgit** [comment]
That fixed it, thank you 🙂
